extensions/wikia/AdEngine/js/template/bfaaMobile.js

```diff
@@ -10,6 +11,7 @@ define('ext.wikia.adEngine.template.bfaaMobile', [
 	require.optional('ext.wikia.adEngine.mobile.mercuryListener')
 ], function (uapContext,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 9 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)